cat-home/src/test/java/com/dianping/cat/report/task/TaskHelperTest.java

```diff
@@ -22,23 +22,35 @@
 import java.util.Date;
 
 import junit.framework.Assert;
+
 import org.junit.After;
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
 
+@RunWith(PowerMockRunner.class)
 public class TaskHelperTest {
 
+	@Rule
+	public final ExpectedException thrown = ExpectedException.none();
+
 	@Before
 	public void setUp() throws Exception {
 	}
 
 	@After
 	public void tearDown() throws Exception {
-	}
+    }
 
 	@Test
 	public void testJoinIntArrayChar() {
-		Assert.assertEquals("1,2", TaskHelper.join(new int[] { 1, 2 }, ','));
+		Assert.assertEquals("1,2", TaskHelper.join(new int[]{1, 2}, ','));
```

Comment:
This is an existing test which we are not adding to, so I don't think we should be modifying the style.